tools/autograd/derivatives.yaml

```diff
@@ -1028,7 +1028,7 @@
   indices: non_differentiable
   self: not_implemented("embedding_renorm")
 
-- name: kl_div(Tensor self, Tensor target, int reduction=Mean) -> Tensor
+- name: kl_div(Tensor self, Tensor target, int reduction=Mean, bool log_target=False) -> Tensor
   self: kl_div_backward(grad, self, target, reduction)
```

Comment:
Just pass in `log_target` as an argument to `kl_div_backward`, for example. I don't think buffers are involved.

Reply:
Yeah, figured that out already. Thank you!